src/Client/StubStripeClient.php

```diff
@@ -29,7 +31,7 @@ public function updatePaymentIntent(string $paymentIntentId, array $updateData):
         $this->pause();
     }
 
-    public function confirmPaymentIntent(string $paymentIntentId): Never
+    public function confirmPaymentIntent(string $paymentIntentId, array $params = []): Never
     {
         throw new \Exception("Confirm Payment Intent not implemented in stub - not currently used in load tests");
```

Comment:
This seems wrong, I think? The previous logic included inline simulation of card brand etc., and we know that load tests _do_ now simulate the new server-side confirmation approach.

Reply:
Thanks, fixed in later PR.